### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

**Comment by stejskalleos** [comment]
@nofaralfasi review please

File: Gemfile [hunk 1]
```diff
@@ -2,3 +2,4 @@ source "http://rubygems.org"
 
 gemspec
 
+gem 'theforeman-rubocop', '~> 0.1'
```
Comment (ekohl) [inline Gemfile]: semver says all .y versions in the 0.y series can be breaking so they suggest to pin to:
```suggestion
gem 'theforeman-rubocop', '~> 0.1.0'
```

File: .rubocop.yml
Comment (ekohl) [inline .rubocop.yml:1]: If you really want to benefit from theforeman-rubocop you should also inherit its rules: https://github.com/theforeman/theforeman-rubocop#usage

**Review by ekohl — COMMENTED** [review]
I've asked @archanaserver to look at updating theforeman-rubocop to a more recent version. Enabling this in CI is a good first step.

**Comment by stejskalleos** [comment]
Ruby tests are changing probably due to some changes in the `foreman:develop`, need to investigate further. But rubocop should be set correctly now

**Review by ekohl — APPROVED** [review]
What's your plan with the TODO file?

**Comment by stejskalleos** [comment]
> What's your plan with the TODO file?

1. Fix failing Ruby tests
2. Fix failing foreman_discovery job
3. Apply safe autocorrect suggestions
4. Apply the rest of the autocorrect suggestions
5. ...
6. profit

**Pull request merged**